gnome-extra/frogr/metadata.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<use>
```

Comment:
This file should be filled with info and not be empty. Are you willing to maintain this package?

Reply:
yes